scanning/scanner.py

```diff
@@ -45,7 +45,7 @@ def scan_sites(self, count: int, urls: Union[List[str], None] = None, skip: int
             for site in sites:
                 url = site
                 if isinstance(site, majestic_million.MajesticMillionSite):
-                    url = 'http://{}'.format(site.domain)
+                    url = site.domain
```

Comment:
Mhh, doesn't this trigger the "no scheme was given" warning for every site that is scanned?

Reply:
Yes, right, this would issue a warning for each URL without an explicitly given scheme. This is indeed quite noisy, but in my opinion an important information as the result can vary depending on the scheme (e.g. broken redirection from http to https, or the https version is broken because of invalid certificate or cipher suite).
The use can control the noisyness with the logging level. Maybe it would be more suitable to use logging level INFO instead of WARNING here, but given the importance I opted for a warning.
Another solution would be an option for `WebsiteAnalyzer` to explicitly silence this message (e.g. `no_scheme_warning = False`) which could be controlled by a command line flag.

Reply:
But what is the reason why would make this change to the scanner? I see no reason for it, especially as this concers the majestic scanning. The only effect of this particular change is that
* the prepending of `http://` does not happen in scanner.py, but
* it is done by the WebsiteAnalyzer -- which triggers a warning.

As it is non-optional for majestic sites anyway, I see no reason not to prepend the scheme in the scanner.

Reply:
Simply assuming http distorts the scanning results (for the mentioned reasons) in contrast to a more specific list with given schemes, but I see that this is simply not possible for the Majestic Million list, therefore I'm also fine with simply dropping the changes in `scanner.py`.

Reply:
Yes, I indeed agree that this assumption is not ok for the `--urls-from-file` parameter: if URLs from that file do not specify the scheme, it is added by the website analyzer.

Could you please amend your commit accordingly (i.e., dropping the change in `scanner.py`)?